app/models/billing-info.js

```diff
@@ -11,6 +11,7 @@ export default Model.extend({
   zipCode: attr('string'),
   country: attr('string'),
   vatId: attr(),
+  billingEmail: attr(),
```

Comment:
I'd suggest to explicitly define value type:
```suggestion
  billingEmail: attr('string'),
```